pkg/sdk/testint/event_tables_integration_test.go

```diff
@@ -0,0 +1,227 @@
+package testint
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_EventTables(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	databaseTest, databaseCleanup := createDatabase(t, client)
```

Comment:
we have common database, schema and warehouse for all integration tests. Please use them.

Reply:
fixed